client/src/App.jsx

```diff
@@ -235,58 +234,173 @@ class App extends React.Component {
     }
   }
 
-  render() {
-    console.log('app rerendeing');
+  renderView() {
     return (
-        <div className="container-fluid">
-          <div className="row">
-            <div className="col-12-sm" id="titlebar">
-              <h1 id="titlefont">infoMapp</h1>
-              <h2 id="subtitlefont">Learn More With Just A Click</h2>
+      <div className="container-fluid">
+        <div className="row">
+          <div className="col-12-sm" id="titlebar">
+            <h1 id="titlefont">infoMapp</h1>
```

Comment:
FYI, html/css usually uses skewer-case. Not a dealbreaker, but something to consider for future.